### ISSUE TITLE
Error when using conventional-changelog in existing project

### ISSUE THREAD

**Issue by richabharti1** [release-it/conventional-changelog#25]
I am able to make a release without a conventional-changelog but when I add conventional changelog to a git repo, I get the below error. This error goes away if I remove `.git` folder and create a new repository.

```
λ npm run release

> v3-extension-react-template@0.0.73 release D:\work\temp3
> dotenv release-it --

events.js:353
      throw er; // Unhandled 'error' event
      ^

Error: fatal: ambiguous argument 'v4.0.1..HEAD': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'

    at Transform._transform (D:\work\temp3\node_modules\git-raw-commits\index.js:83:30)
    at Transform._read (D:\work\temp3\node_modules\through2\node_modules\readable-stream\lib\_stream_transform.js:177:10)
    at Transform._write (D:\work\temp3\node_modules\through2\node_modules\readable-stream\lib\_stream_transform.js:164:83)
    at doWrite (D:\work\temp3\node_modules\through2\node_modules\readable-stream\lib\_stream_writable.js:409:139)
    at writeOrBuffer (D:\work\temp3\node_modules\through2\node_modules\readable-stream\lib\_stream_writable.js:398:5)
    at Transform.Writable.write (D:\work\temp3\node_modules\through2\node_modules\readable-stream\lib\_stream_writable.js:307:11)
    at Socket.ondata (internal/streams/readable.js:745:22)
    at Socket.emit (events.js:388:22)
    at addChunk (internal/streams/readable.js:309:12)
    at readableAddChunk (internal/streams/readable.js:280:11)
Emitted 'error' event on Readable instance at:
    at Transform._transform (D:\work\temp3\node_modules\git-raw-commits\index.js:83:16)
    at Transform._read (D:\work\temp3\node_modules\through2\node_modules\readable-stream\lib\_stream_transform.js:177:10)
    [... lines matching original stack trace ...]
    at readableAddChunk (internal/streams/readable.js:280:11)
```
In release-it.js I have written the following code:
```'plugins': {
        '@release-it/conventional-changelog': {
            'preset': 'angular',
            'infile': 'CHANGELOG.md',
            'ignoreRecommendedBump': true,
        },
    }
```

**Comment by webpro** [comment]
Probably `v4.0.1` was not a tag in the repo. Not sure where it came from, though. Can you provide any more details?

**Comment by onikienko** [comment]
In case someone will have a similar issue. 

We worked together with @richabharti1 on this issue. We found that in our git log we have one commit message which contains the words "tag: v4.0.1" in the message body. It's just to mention the tag from another repo.  For later reference.  

Looks like `git-raw-commits` tries to find this tag in our repo. And crashes on this step because we do not have such a tag. After we changed this commit message to do not have "tag: v4.0.1"  and `git push --force`, we created a release with conventional-changelog as expected. 

Thanks, @webpro for all of your work!
